_posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Ranking Points & Cosmetic XP
+
+permalink: /guide/ranking-cosmetic-xp/
+
+category: guide
+
+tags:
+  - rank
+  - xp
+  - experience
+  - cosmetic
+
+toc: true
+
+toc_sticky: true
+---
+
+# Ranked-Points System
+
+**The points you earn from your time on a map is the sum of the points awarded from each of these 3 systems described below: WR/top10 points + formula points + group points.**
+
+{:.notice--info}
+The systems described here are complete; we are in agreement that this is what the Ranked-Points should look like. The values chosen for percentiles, WR points, etc. are still up for debate; however, these should be simple to change later, should we decide to change them, because of the design of the system. For now, we consider the values to be informed-guess placeholders.
+
+> There are no “base completion points”; the formula system somewhat accounts for this, but we’ve determined that it is not worth having a system of its own
+
+<sub>Aside -- Motivation;</sub>
+
+<sub>Times ranked beyond the top10 are awarded points according to 2 systems: the Group system, and a Formula system. The group system rewards discrete bundles of points for players ranking within certain “percentile” blocks. The formula system awards points simply according to a formula with a single parameter -- the player’s rank.
+The group system should award more points than the formula system; it should be the primary goal of the player to improve to the next group. The formula system simply provides bonus ranked points for improving within a group, so the player does not feel like intermediate improvements are a waste of effort.</sub>
+
+
+## WRs and Top10s system
+
+WRs award a fixed number of points for all maps. For now, we set this to be 3,000 points.
+Ranks 2-10 award some percentage of the WR points.
+
+```
+Rank 1 = 3000 points = 100% of WR
+Rank 2 = 2250 points = 75% of WR
+Rank 3 = 2040 points = 68% of WR
+Rank 4 = 1830 points = 61% of WR
+Rank 5 = 1710 points = 57% of WR
+Rank 6 = 1590 points = 53% of WR
+Rank 7 = 1515 points = 50.5% of WR
+Rank 8 = 1440 points = 48% of WR
+Rank 9 = 1365 points = 45.5% of WR
+Rank 10 = 1290 points = 43% of WR
+```
+
+## Formula system
+
+Points awarded is some equation of the form `A / (rank + B)`, where `A` is a scaling constant, and `B` is a smoothing constant.\
+We choose `A` = 50000, `B` = 49.\
+In this case the graph shows our equation `Points = 50000 / (rank + 49)`\
+![Formula Graph](https://i.imgur.com/wjoJYNg.png)
+
+{:.notice--info}
+X axis designates player rank on a map,\
+y axis designates points awarded
+
+<sub>Ex:\
+Player sets a time of rank 468 on a map with 1333 completions.\
```

Comment:
Note that this renders as
![image](https://user-images.githubusercontent.com/9014762/154193788-7b168575-3e44-490e-9248-4e563b41c36b.png)

Will have to use `<br>` when inside an html tag I believe. There are other instances of this in this guide as well.